### PR TITLE
Handle case where ora2 score override had no previous score

### PR DESCRIPTION
**Pull request by kluo**
@dcadams @stvstnfrd

### CODE REVIEW

File: scripts/migrate_score_override.py [hunk 1]
```diff
@@ -29,12 +29,21 @@ def main():
...
                     created_at__lte=score.created_at,
                     submission__isnull=False,
-                ).order_by('-created_at')[:1].get().submission
+                ).order_by('-created_at')[:1]
+                if preceding_score.count():
+                    submission = preceding_score.get().submission
+                else:
+                    submission_qset = Submission.objects.filter(student_item=score.student_item)
+                    if submission_qset.count() > 1:
+                        raise Exception("MULTIPLE SUBMISSIONS FOR STUDENT_ITEM {}".format(score.student_item))
```
Comment (stvstnfrd) [inline scripts/migrate_score_override.py]: Do we actually want to bail completely on the entire loop?
Should we log the failure and continue onto the next override?

**Comment by stvstnfrd** [comment]
@kluo What's the status here?

File: scripts/migrate_score_override.py (hunk 1 above)
Comment (kluo) [reply scripts/migrate_score_override.py]: No since there can be multiple overrides for the same student_item, don't want to have it finish out of order

**Comment by kluo** [comment]
Ready to merge with +1

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**